### ISSUE TITLE
grading ass1

### ISSUE THREAD

**Issue by gosiamigut** [MarikWesterhof/Machine-Learning#1]
Total: 11/15
Programming assignment: 7.5/10
- well done
- oeps! the cost is not the amount of correctly classified examples.... That you should have calculated by comparing your predition to the label and counting
- your cost analysis would be good if your would call it cost analysis and not best score analylsis.
- do you overfit or not on your data set?
- no analysis of the learning rate (always report the results)

Written assignment: 3.5/5
see the correct answers at https://staff.fnwi.uva.nl/m.a.migut/AUC_ML/solutions.html
